spec/features/families/admin_views_all_families_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+feature "Admin can see all the families when in families#index" do
+  context "when click on the 'All' button" do
+    scenario "successfully" do
+      create(:family, active: false, last_name: "Scara")
+      create(:family, active: true, last_name: "Liso")
```

Comment:
Consider adding a trait in the family factory, so that you can just use:
`create(:family, :active)`

Reply:
Modified in sha `b6c4d17`